sdk/cargo-build-sbf/src/main.rs

```diff
@@ -55,6 +57,7 @@ impl Default for Config<'_> {
             no_default_features: false,
             offline: false,
             remap_cwd: true,
+            debug: true,
```

Comment:
Maybe at least until sbfv2 we should have it `false` by default. And change the command line option to `enable-debug` instead.

Reply:
@dmakarov Good point, fixed. I called the flag `--debug`. When it becomes default it can be disabled with `--debug=false`. I think that sounds more natural or should I rename it to `--enable-debug`?

Reply:
`—debug` is fine

Reply:
the default still seems to be `true`, however.

Reply:
Sorry about that, fixed now.